tests/tests_geomstats/test_datasets.py

```diff
@@ -10,6 +10,7 @@
 from geomstats.geometry.spd_matrices import SPDMatrices
 from geomstats.geometry.special_euclidean import SpecialEuclidean
 from geomstats.geometry.special_orthogonal import SpecialOrthogonal
+from geomstats.geometry.discrete_curves import R2, DiscreteCurves
```

Comment:
Imports should be sorted in alphabetic order:
`geomstats.geometry.discrete_curves` comes before `from geomstats.geometry.euclidean`.

Reply:
And DiscreteCurves comes before R2